### DISCUSSION TITLE
大家認為Google有壟斷之嫌嗎？

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#206] [General]
幾年以前就已經傳出google被反壟斷法盯上的消息，其優秀的市佔率和整包帶走的服務型態使它成為最具爭議的科技公司之一。而各國家多少都有反壟斷的法條存在，google是否有壟斷，或是要怎麼保護自己不受法律找碴呢？

新聞連結：避免遭德國反壟斷調查 谷歌將自搜索結果移除新聞服務
http://m.cnyes.com/news/id/4802340?utm_source=App&utm_medium=android&utm_campaign=4802340

**Comment by handsomeniko** [comment]
以前確實有感覺，但我覺得這都是會改變的，慢慢的微軟其實也在擴大瀏覽器這塊市場。

**Reply by ayakasanma** [reply]
EDGE真心好用~Chrome佔用太多記憶體這件事遲遲沒改善，總有一天會翻船被微軟取而代之!
(但是Bing搜尋一樣悲劇=.=)

**Comment by Shiba010** [comment]
從以前到現在，感覺google越來越厲害，快要沒有對手了...

**Comment by 110409538** [comment]
Google在許多人的生活中是不可或缺的

**Comment by cobb110202008** [comment]
市場上還是需要避免有壟斷的情形發生，多元的公司互相競爭才是良好的市場狀態。

**Comment by Yunhans** [comment]
有覺得，但是還是會用

**Comment by LanningYang** [comment]
確實有感覺，但其家族的相關連性，使得在日常生活中增加方便性。

**Comment by lx3211** [comment]
小時候都還會用yahoo搜尋東西
長大就完全不會了呢

**Comment by xCk27x** [comment]
像我就是用edge作為主力瀏覽器 
因為有一些功能覺得很有趣就果斷跳槽了
用到現在也沒有甚麼問題
>>>
但就算egde等其他瀏覽器功能漸漸強大
使用者換瀏覽器的成本還是太高了
所以短期內應該不會有太多變化

**Comment by NcuJerryKao** [comment]
google 及 youtube 都有壟斷吧
現在要看影片 第一個想法就是在YT找

**Comment by Ben6uo** [comment]
我不認為有壟斷 只是google的技術更加的熟而已

**Comment by gnsJhenJie** [comment]
感覺大部分科技公司都很喜歡玩套養殺這招，像是Google 原本提供學校無限容量的Google Workspace帳號，去年突然公布取消，大家都已經習慣使用Google Drive、Google Meet等軟體了，但開始收費後，要像過去一樣繼續使用就要花高達數百外的價錢才行。
另外，Netflix過去都默許共用帳號的狀況，但最近開始抓這些共用的狀況，也是類似的概念。不過我認為單就這兩個狀況來說，沒有到壟斷這麼嚴重。

**Reply by chia-kai-chang** [reply]
@gnsJhenJie 助教的是每一篇都認真發文~ 感動 😂

**Comment by BingRong1217** [comment]
但google已經成為生活中不可或缺的一部份了

**Comment by awkward-willy** [comment]
縱使有壟斷之嫌，這類型的服務確實得要有一套完整的產品方能完整發揮它的功能，故我反而認為一條龍的 Google 服務是很方便且快速的。

**Comment by limitx0** [comment]
Google的服務很全面也很好用
雖然可能有壟斷但真的很方便

**Comment by Yilin20021016** [comment]
如果沒有那麼好的東西 那裡會有比他好的出現?壟斷只是暫時的

**Comment by Raywei0214** [comment]
我覺得主要還是看個人習慣吧，用久了去用其他瀏覽器反倒會覺得不太適應，攏不壟斷什麼的我覺得就是多餘的了，就像你平常洗澡都用沐浴乳，今天突然只能用肥皂，或多或少會有點不習慣吧，要用哪種瀏覽器就屬於個人行為吧

**Comment by magicmicheal** [comment]
用完Google，再回去用Yahoo，整個人很躁，壟斷不壟斷握認為是沒有的，google又沒花錢吞併小公司，要怪只能怪小公司的服務不夠全面，無法讓使用者變心

**Comment by larry40305** [comment]
我覺得微軟有漸漸跟上Google，像現在其實滿多人使用Edge。
